tests/test_pipdeptree.py

```diff
@@ -1,13 +1,20 @@
 from __future__ import annotations
 
 import sys
-from pathlib import Path
+from importlib.metadata import Distribution
 from subprocess import check_call  # noqa: S404
 
 
 def test_main() -> None:
     check_call([sys.executable, "-m", "pipdeptree", "--help"])
 
 
-def test_console() -> None:
-    check_call([Path(sys.executable).parent / "pipdeptree", "--help"])
+def test_console(mocker: MockerFixture) -> None:
+    d = Distribution.from_name("pipdeptree")
+    our_entry_points = d.entry_points
+    assert len(our_entry_points) == 1
+    pipdeptree_entry_point = next(iter(our_entry_points))
+    pipdeptree_module = pipdeptree_entry_point.load()
+    mocker.patch("pipdeptree.__main__.sys.argv", [""])
```

Comment:
It looks args will be passed to pipdeptree if not empty them.
```
message = '__main__.py: error: unrecognized arguments: --color=yes --cov /data/upstream/cunshunxia/pipdeptree/.tox/py311/lib/pyt...hunxia/pipdeptree/.tox/coverage.py311.xml --junitxml /data/upstream/cunshunxia/pipdeptree/.tox/junit.py311.xml tests\n'
```

Reply:
Instead of mocking, just do `pipdeptree_module([])` since `load()` is actually having us call our `main()` when it parses `pipdeptree.__main__:main` (`argparse` defaults to using `sys.argv` if we don't pass it anything).

Reply:
Great. Done this.